lib/src/strings.dart

```diff
@@ -186,6 +186,7 @@ const windowsDylibLocations = [
 ];
 const macOsDylibLocations = [
```

Comment:
nit - this could be a `Set` 😄

Reply:
Done. Looks safe to me for windows and linux too:
https://github.com/dart-lang/ffigen/search?q=macOsDylibLocations
https://github.com/dart-lang/ffigen/search?q=windowsDylibLocations
https://github.com/dart-lang/ffigen/search?q=linuxDylibLocations

Reply:
A [LinkedHashSet](https://api.dart.dev/stable/2.13.4/dart-collection/LinkedHashSet-class.html), which a const set is. Because we would not want to make the iteration order undefined and make llvm resolution non-deterministic.

Reply:
Does it really make a difference though? Since we only need to iterate, an array should be the same as a LinkedHashSet in that regard.